lib/std/os.zig

```diff
@@ -5215,6 +5222,49 @@ pub fn getFdPath(fd: fd_t, out_buffer: *[MAX_PATH_BYTES]u8) RealPathError![]u8 {
             };
             return target;
         },
+        .freebsd => {
+            const allocator = std.heap.c_allocator;
+            const mib = [4]c_int{ CTL.KERN, KERN.PROC, KERN.PROC_FILEDESC, system.getpid() };
+
+            // Get an initial size for the kinfo buffer and allocate.
+            // The buffer is used to store the varible-sized kinfo_file structures.
+            // Note that all allocations are 64-bit aligned, as is kinfo_file.
+            var len: usize = 0;
+            try sysctl(&mib, null, &len, null, 0);
+            var buf = try allocator.alignedAlloc(u8, 8, len);
```

Comment:
Since we are allocating u8 instead of u64, we do not need to do the (len / 8) + 1 calculation anymore.